pyworklistserver/worklist_server.py

```diff
@@ -39,6 +41,8 @@ def get_seedNumber(self):
                 f.write(str(seed))
         return seed
 
+=======
+>>>>>>> 335d443 (Put content in user-config.py in dictionary, and updated use througout project. Purpose: Cleaner code and opening the possibility to merge with config-input from command-line.)

```

Comment:
Looks like there are still conflicts that needs to be resolved.

Reply:
Yes, thanks for the noticing. Some changes were not staged and commited before the push. Mistake on my part.